test/solidity-ibc/IntegrationTest.t.sol

```diff
@@ -495,82 +437,6 @@ contract IntegrationTest is Test {
         ics26Router.recvPacket(msgRecvPacket);
     }
 
-    // This test case tests the scenario where IBCStore.setPacketReceipt fails with custom error
```

Comment:
Are there any cases that we are not covering by deleting this? I understand it works differently now, but do we cover the new paths?

Reply:
No, we are not missing anything. This code path simply doesn't exist anymore. Which is good.